calcom/cal.com
# feat: OOO booking-forwarding

## What does this PR do?

New feature that should handle profile redirection or away status if setup by user.

Fixes #11031

https://www.loom.com/share/1774045f98ed4759bf57322e62bf63f4

## Type of change




## How should this be tested?



- Are there environment variables that should be set? No
- What are the minimal test data to have? Seed data, teampro, pro users
- What is expected (happy path) to have (input and output)? When setup we handle redirect on user out of office to selected user.

## Mandatory Tasks
